server/controllers/v1/auth.controller.js

```diff
@@ -1,8 +1,19 @@
 const { request, response } = require('express');
 
+import user from '@models/user.js';
+
 const login = (request, response) => {};
 
-const register = (request, response) => {};
+const register = async (request, response) => {
+  const { name, email, password } = request.body;
```

Comment:
Codacy found an issue: [Unnecessary block.](https://app.codacy.com/manual/sam-spain/full-stack-js/pullRequest?prid=5992206)

Reply:
Doesn't seem applicable in this case.